samples/utils/CommandLineUtils.cpp

```diff
@@ -198,6 +198,52 @@ namespace Utils
             "The password to send when connecting through a custom authorizer (optional)");
     }
 
+    void CommandLineUtils::AddLoggingCommands()
+    {
+        RegisterCommand(
+            m_cmd_verbosity,
+            "<log level>",
+            "The logging level to use. Choices are 'trace', 'debug', 'info', 'warn', 'error', 'fatal', and 'none'. "
```

Comment:
I have to dig around a bit but I feel like these have been "Trace", "Debug", etc w/the first letter capitalized in other SDKs. Might be good to try to be as consistent as possible in the event all the others are this way.